pkgs/development/python-modules/rouge-score/default.nix

```diff
@@ -0,0 +1,60 @@
+{ lib
+, fetchPypi
+, fetchFromGitHub
+, python
+, buildPythonPackage
+, absl-py
+, nltk
+, numpy
+, six
+, pytestCheckHook
+, pythonOlder
+}:
+let
+  testdata = fetchFromGitHub {
+    owner = "google-research";
+    repo = "google-research";
+    sparseCheckout = [ "rouge/testdata" ];
+    rev = "1d4d2f1aa6f2883a790d2ae46a6ee8ab150d8f31";
+    hash = "sha256-ojqk6U2caS7Xz4iGUC9aQVHrKb2QNvMlPuQAL/jJat0=";
+  };
+in buildPythonPackage rec {
+  pname = "rouge_score";
+  version = "0.1.2";
+  format = "setuptools";
+  disabled = pythonOlder "3.7";
+
+  src = fetchPypi {
+    inherit pname version;
+    extension = "tar.gz";
+    hash = "sha256-x9TaJoPmjJq/ATXvkV1jpGZDZm+EjlWKG59+rRf/DwQ=";
+  };
+
+  # the tar file from pypi doesn't come with the test data
+  patchPhase = ''
```

Comment:
It is better to patch with `postPatch` than to override the `patchPhase` behavior, I think.